src/main/resources/com/google/api/codegen/php/sample_common.snip

```diff
@@ -1,223 +0,0 @@
-@snippet methodCallSampleCode(apiMethod)
```

Comment:
Since standalone samples and method samples have pretty much diverged (And may be more different in the future), it seems unnecessary to try to reuse the snippets in `sample_common`. Deleted `sample_common` and let `standalone_sample` and `method_sample` define their own templates. Java and Python follow this pattern too.